src/Example/UserRegistrationBundle/Tests/Controller/UserRegistrationControllerTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/*
+ * Copyright (c) 2012-2014 KUBO Atsuhiro <kubo@iteman.jp>,
+ * All rights reserved.
+ *
+ * This file is part of PHPMentors_Training_Example_Symfony.
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the BSD 2-Clause License which accompanies this
+ * distribution, and is available at http://opensource.org/licenses/BSD-2-Clause
+ */
+
+namespace Example\UserRegistrationBundle\Tests\Controller;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\BrowserKit\Client;
+use Symfony\Component\Validator\ConstraintValidatorInterface;
+
+use Example\UserRegistrationBundle\Event\BundleEvent;
+
+class UserRegistrationControllerTest extends WebTestCase
+{
+    /**
+     * @return EntityManagerInterface
+     */
+    public function createEntityManager()
+    {
+        $entityManager = \Phake::mock('Doctrine\ORM\EntityManager');
+        $metadataFactory = \Phake::mock('Doctrine\Common\Persistence\Mapping\AbstractClassMetadataFactory');
```

Comment:
__[development][phpmd]__
* Avoid using static access to class '\Phake' in method 'createEntityManager'.